LibreNMS/OS/Gigamon.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Gigavue.php
+ *
+ * Gigamon GigaVUE OS
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2020 Tony Murray
```

Comment:
Change this to yours

Reply:
Waiting for confirmation as the initial code I based this on is basically exactly his. Once I hear from him I will change this and the test files.